classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label, students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students << (student)
+    student.classroom = self
+  end
+end
```

Comment:
- [ ]  When setting the classroom for a student it also adds it to the classrooms' students. 👇🏻 


![image](https://user-images.githubusercontent.com/79058364/223626079-a45a2e43-1163-4413-b293-2fb9de1c4cdf.png)

- According to the project requirements, 🤔  You should implement a method that serves as the 'belongs-to' side (a student belongs to a classroom) ✍🏻 

- Your code may look like this 👇🏻

```ruby
class Student < Person
  attr_reader :classroom
  .
  def classroom=(classroom)
      ...
  end
end
```